include/wx/qt/private/treeitemdelegate.h

```diff
@@ -67,6 +68,28 @@ class wxQTTreeItemDelegate : public QStyledItemDelegate
         QStyledItemDelegate::setModelData(editor, model, index);
     }
 
+    bool helpEvent(QHelpEvent *event, QAbstractItemView *view, const QStyleOptionViewItem &option, const QModelIndex &index)
+    {
+        if ( event->type() == QEvent::ToolTip)
+        {
+            const QRect itemRect = view->visualRect(index);
+            const QSize bestSize = sizeHint(option, index);
+            if ( itemRect.width() < bestSize.width() )
+            {
+                QString value = index.data(Qt::DisplayRole).toString();
+                QToolTip::showText(event->globalPos(), value, view);
+            }
+            else
+            {
+                QToolTip::hideText();
```

Comment:
Is this really needed? Currently is the tooltip always shown?

Reply:
I'll retest this,  but I seem to remember that without this the tooltip remained displayed in incorrect situations.